src/webgl/loading.js

```diff
@@ -229,18 +229,16 @@ p5.prototype.loadModel = function(path,options) {
           const parsedMaterials=await getMaterials(lines);
 
           if(parsedMaterials && typeof parsedMaterials==='object'){
-            try {
-              parseObj(model, lines, parsedMaterials);
-            } catch (error) {
-              console.error(error.message);
-            }
+            parseObj(model, lines, parsedMaterials);
           }
         }catch (error) {
+          console.error(error.message);
```

Comment:
Do we need to `console.error` here when we `_friendlyError` below?

Reply:
Ah yes! I thought of it but was only checking all errors in console while making this work. Sorry for the oversight while pushing. I'll change these to Friendly Errors.